pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -7,6 +7,46 @@
             "#import": [ "base" ]
         },
 
+        "android": {
+            "#import": [ "any" ]
+        },
+        "android-arm64": {
+	           "#import": [ "any" ]
+        },
+        "android-mips": {
+            "#import": [ "any" ]
+        },
+
+        "android.21": {
+	           "#import": [ "android" ]
+	       },
+        "android.21-arm64": {
+	           "#import": [ "android.21", "android-arm64" ]
+        },
+        "android.21-mips": {
+	           "#import": [ "android.21", "android-mips" ]
+        },
+
+        "android.22": {
```

Comment:
From an ABI point of view, the different versions are very backwards compatible. The package management system we've used to cross-compile for Android, Termux, only compiles for API level 21 at this moment - and you can take those binaries and run them up to API level 25, which is the most recent version available.

So I'm thinking we should either add API level 21 through 25, or only 21.

I don't know where MIPS comes from - I think the idea was to add arm.
At this time, CoreCLR supports x86, x64, arm and arm64 - there is no support for MIPS on any platform. Adding MIPS support would, amongst others, require porting [assembly code](https://github.com/dotnet/coreclr/tree/master/src/vm/amd64) to MIPS, I don't think anyone is working on it.

Net, I would suggest:
- Replacing MIPS with arm
- Adding only API level 21, or 21 through 25.